src/store/mapContainer.ts

```diff
@@ -61,7 +60,25 @@ const parallelLimit = <T>(
     return Promise.all(arrChains).then(() => result);
 };
 
-export const getLightnode = (network: RenNetworkDetails): string => {
+export const getLightnode = (
+    network: RenNetworkDetails,
+    isNew = false,
+): string => {
+    if (isNew) {
+        switch (network.name) {
+            case "mainnet":
+                return "https://lightnode-new-mainnet.herokuapp.com";
```

Comment:
Since the v0.4 update, we can leave out the `-new` since they point to the same lightnode.